cfgrammar/src/lib/yacc/ast.rs

```diff
@@ -195,7 +209,7 @@ impl fmt::Display for Symbol {
 }
 
 impl GrammarAST {
-    pub fn new() -> GrammarAST {
+    pub(crate) fn new() -> GrammarAST {
```

Comment:
This change of visibility was only included in this patch accidentally, I feel like it is good to consider whether we should do this, but probably doesn't belong in this series.

So I'd rather drop it from this PR.